modules/client-configuration/pages/channels.adoc

```diff
@@ -33,6 +33,12 @@ This provides the {phub} channels, which you can subscribe your clients to.
 
 On Salt clients, you must apply the highstate after subscribing to the channel to be able to install packages.
 
+[NOTE]
+====
+{phub} provides a large number of packages, which can take a long time to synchronize.
+Do not enable it unless you require the packages it provides.
```

Comment:
```suggestion
To avoid unintended updates or installations of software from a non-SUSE supported source, it is recommended to establish a Content Lifecycle Management strategy based on denying all {phub} packages first and then explicitly enabling only the specific packages you require.
```